staging/src/k8s.io/dynamic-resource-allocation/structured/internal/incubating/pools_incubating.go

```diff
@@ -52,6 +52,7 @@ func nodeMatches(node *v1.Node, nodeNameToMatch string, allNodesMatch bool, node
 // Both is recorded in the result.
 func GatherPools(ctx context.Context, slices []*resourceapi.ResourceSlice, node *v1.Node, features Features) ([]*Pool, error) {
 	pools := make(map[PoolID]*Pool)
+	slicesWithBindingConditions := []*resourceapi.ResourceSlice{}
```

Comment:
Don't need to initialize here:
```suggestion
	var slicesWithBindingConditions []*resourceapi.ResourceSlice
```